tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -555,7 +597,9 @@ async def test_recovering_view_after_restart_with_packet_loss(self, bft_network,
     @staticmethod
     async def _await_replicas_in_state_transfer(logger, bft_network, skvbc, primary):
         for r in bft_network.get_live_replicas():
+            logger.log_message("got live replicas")
```

Comment:
I would recommend for better logging to get all the live replicas before the loop and print them. 
for example:
`        
        live_replicas = bft_network.get_live_replicas()
        logger.log_message(f"live replicas = {live_replicas}")
        for r in live_replicas:
`